### ISSUE TITLE
Bounding Boxes Example

### ISSUE THREAD

**Issue by MentalGear** [albu/albumentations#323]
## 📚 Documentation
Very interesting library, though it would be great if we could have an example on how to use if we need Bounding Box support. Espeically, if we want to retain the label(id) of the bounding box.

Example: Picture with a bootle and a glass. How does one not only specifiy the coordinates of the BB but also the names ?

**Comment by albu** [comment]
https://github.com/albu/albumentations/blob/master/notebooks/example_bboxes.ipynb

**Comment by MentalGear** [comment]
Hi @albu 
Thanks for the quick reply!
It seems there's an issue when installing from PyPI, as using
!pip install albumentations

gave me the following error:
```
ImportError 
<ipython-input-3-a7c310633abd> in <module>()
----> 1 from albumentations import (
      2     BboxParams,
      3     HorizontalFlip,
      4     VerticalFlip,
      5     Resize,

ImportError: cannot import name 'BboxParams'
```

Afterwards, I used the
!pip install -U git+https://github.com/albu/albumentations
directly and it worked just fine !

So just a head's up for you, as there's either a bug with BboxParams in the version on PyPI or maybe BboxParams is just new feature ?

**Comment by albu** [comment]
oh sorry, we just merged BboxParams and not yet updated it in pip. thanks for pointing!

**Comment by albu** [comment]
fixed in 0.3.2
